caliban/util.py

```diff
@@ -402,19 +403,23 @@ def capture_stdout(cmd: List[str],
 
   buf = io.StringIO()
   ret_code = None
+
   with subprocess.Popen(cmd,
                         stdin=subprocess.PIPE,
                         stdout=subprocess.PIPE,
                         stderr=subprocess.STDOUT,
-                        bufsize=1,
-                        encoding="utf-8") as p:
+                        universal_newlines=False,
+                        bufsize=1) as p:
     if input_str:
-      p.stdin.write(input_str)
```

Comment:
TextIOWrapper below doesn't like the encoding, so I removed it from the constructor above and manually apply it here on stdin.